core/src/main/resources/messages/messages.properties

```diff
@@ -47,7 +47,7 @@ error.buddy.user.cannot.be.null=User cannot be null
 error.buddy.requesting.user.cannot.be.null=Requesting user cannot be null
 error.buddy.action.unknown=Action ''{0}'' is unknown
 error.buddy.message.entity.cannot.be.null=Message entity cannot be null
-error.buddy.vpnloginid.cannot.be.null=VPN Login Id cannot be null 
+error.buddy.useranonymizedid.cannot.be.null=Anonymized user id cannot be null 
```

Comment:
All the messages have id in lowercase. It's more consistent now ;). Do you want me to do a global replace?